.config/pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts = -v -rf --ff --cov=geoips --cov-report=term-missing
```

Comment:
Remove --cov=geoips from the pytest.ini, pass in --cov=<pkg> from command line (since we can run fro multiple packages)